### DISCUSSION TITLE
NEO NAS-WV03B2 (_TZE204_4fblxpma, TS0601) Smart Irrigation Valve

### DISCUSSION THREAD

**Discussion by bbadrignans** [zigpy/zha-device-handlers#3182] [Q&A]
I need some help to develop the quirk for the NEO NAS-WV03B2 irrigation timer valve, with the vendor ID _TZE204_4fblxpma and model TS0601.
![image](https://github.com/zigpy/zha-device-handlers/assets/534410/2b2280f7-8478-4e49-8376-0b730d249337)

https://www.szneo.com/en/products/show.php?id=273

I could find in ZHA that there is a quirk that support TS0601 valves : https://github.com/zigpy/zha-device-handlers/issues/1556

Changing only the VID in [ts0601_valve.py](https://github.com/zigpy/zha-device-handlers/blob/dev/zhaquirks/tuya/ts0601_valve.py) didn't work well. I tried for all the valve supported.
The best result is : valve states is correctly reported to HA but I can't set valve state from HA.

I haven't a Tuya gateway which make it difficult to find the good DP.
Does anyone got the same devices and can help me ?

**Comment by Hedda** [comment]
If you do not want help developming or someone else to develop such a “quirk” Python script yourself, you can submit a “device support request” as a new issue to this GitHub repository instead of posting here in discussions:

https://www.home-assistant.io/integrations/zha/#how-to-add-support-for-new-and-unsupported-devices

**Reply by bbadrignans** [reply]
Thank you for the suggestion, just made it here : https://github.com/zigpy/zha-device-handlers/issues/3204

Just find out this model was added in https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/giex.ts few days ago.
Anyone knows how to adapt from zigbee-herdsman-converters to zha quirks ?
